src/main.ts

```diff
@@ -3,7 +3,8 @@ import {
   requireEvent,
   requireFiles,
   requirePr,
-  requirePullNumber
+  requirePullNumber,
+  require_cc0
 } from "#/assertions";
```

Comment:
Did you make sure to add these to https://github.com/JEAlfonsoP/EIP-Bot/blob/master/src/modules/assertions/index.ts?

Reply:
@JEAlfonsoP I don't think you meant to mark this as resolved.

Reply:
I did not ! Actually I did have a question about it: assertions/index.ts refers to index type  and not setting the default export for the function, ideal is to set all functions as not default or do you have the criteria for chosen it who goes to index.ts ?

Reply:
done